### ISSUE TITLE
[🐛BUG] widedeep的AUC性能与论文有偏差  ml-1m数据集

### ISSUE THREAD

**Issue by NEUYuYang** [RUCAIBox/RecBole#2166]
以下是我的参数
# dataset config
gpu: 0
field_separator: "\t" #指定数据集field的分隔符
seq_separator: " " #指定数据集中token_seq或者float_seq域里的分隔符
USER_ID_FIELD: user_id #指定用户id域
ITEM_ID_FIELD: item_id #指定物品id域
RATING_FIELD: rating #指定打分rating域
TIME_FIELD: timestamp #指定时间域
NEG_PREFIX: neg_ #指定负采样前缀
LABEL_FIELD: label #指定标签域
threshold:
    rating: 4
ITEM_LIST_LENGTH_FIELD: item_length #指定序列长度域
LIST_SUFFIX: _list #指定序列前缀
MAX_ITEM_LIST_LENGTH: 50 #指定最大序列长度
POSITION_FIELD: position_id #指定生成的序列位置id
#指定从什么文件里读什么列，这里就是从ml-1m.inter里面读取user_id, item_id, rating, timestamp这四列,剩下的以此类推
load_col:
    inter: [user_id, item_id, rating, timestamp]
    user: [user_id, age, gender, occupation]
    item: [item_id, genre]

# training settings
epochs: 500 #训练的最大轮数
train_batch_size: 1024 #训练的batch_size
learner: adam #使用的pytorch内置优化器
train_neg_sample_args: ~ #负采样数目
eval_step: 1 #每次训练后做evalaution的次数
stopping_step: 10 #控制训练收敛的步骤数，在该步骤数内若选取的评测标准没有什么变化，就可以提前停止了

# evalution settings
eval_args:
  split: {'RS': [0.8,0.1,0.1]} #对数据随机重排，设置按比例划分数据集
  group_by: ~ #是否将一个user的记录划到一个组里
  mode: labeled
  order: RO
metrics: ['AUC'] #评测标准
group_by_user: False
valid_metric: AUC #选取哪个评测标准作为作为提前停止训练的标准
eval_batch_size: 1024 #评测的batch_size

show_progress: True

val_interval:
    rating: "[3,inf)"    

user_inter_num_interval: "[10,inf)"
item_inter_num_interval: "[10,inf)"
mlp_hidden_size: [32, 16, 8]    # (list of int) The hidden size of MLP layers.

以下为run_hyper参数
embedding_size choice [16,32,64]
learning_rate choice [0.1,0.01,0.05,0.001, 0.005, 0.0001]
dropout_prob choice [0.1, 0.2, 0.3]

当我用这些参数跑时，最佳AUC只能达到0.7607  而在一些论文中，AUC可以达到0.9
例如STEC: SEE-THROUGH TRANSFORMER-BASED ENCODER FOR CTR PREDICTION
An End-to-End Neighborhood-based Interaction Model for Knowledge-enhanced Recommendation
RippleNet: Propagating User Preferences on the Knowledge Graph for Recommender Systems
Multi-Task Feature Learning for Knowledge Graph Enhanced Recommendation
DCNv3: Towards Next Generation Deep Cross Network for Click-Through Rate Prediction
请问这是为什么

**Comment by TayTroye** [comment]
@NEUYuYang 你好!
对于ml数据集一般采用以下的方式加载特征，并且无需过滤掉小于10的交互。
```
load_col:
   inter: [user_id, item_id, rating]
   user: [user_id, age, gender, occupation]
   item: [item_id, genre, release_year]
```
你可以增大`mlp_hidden_size`为`[256,256,256]`来提高模型性能
